### PR TITLE
FOLSPRINGB-36: Add postgres dependency to pom.xml

### PR DESCRIPTION
**Pull request by librarianmage**
https://issues.folio.org/projects/FOLSPRINGB/issues/FOLSPRINGB-36

## Purpose
In the base, Postgres is a dependency but it is not loaded by `pom.xml`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,6 +85,11 @@
       <artifactId>lombok</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${postgresql.version}</version>
```
Comment (psmagin) [inline pom.xml:91]:
```suggestion
      <scope>runtime</scope>
```
The version of postgresql should be in sync with the spring libs that working with it. So I propose to use the dependency that `spring-boot-starter-parent` provides.
Also, we can use `runtime` scope for PostgreSQL driver.

**Review by psmagin — CHANGES_REQUESTED** [review]

**Comment by librarianmage** [comment]
Makes sense to me!

**Review by psmagin — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
     <java.version>11</java.version>
     <spring-cloud-starter-openfeign.version>3.1.0</spring-cloud-starter-openfeign.version>
     <commons-lang3.version>3.12.0</commons-lang3.version>
+    <postgresql.version>42.3.1</postgresql.version>
```
Comment (psmagin) [inline pom.xml:22]: So this property is not needed now.

**Comment by librarianmage** [comment]
Hmm, not sure why it decided that it's making all of these changes.

**Comment by ncovercash** [comment]
@psmagin @dmtkachenko this should be ready for review now!

**Comment by psmagin** [comment]
I suggest to merge this in master, not in release branch.

**Comment by ncovercash** [comment]
@psmagin not sure how that happened, but it seems we can't change it.  Should re rebase and make a new PR?

**Comment by julianladisch** [comment]
@librarianmage Thank you for your contribution! Your commits have been merged to master: #49

Therefore this pull request is no longer needed and can be closed.

**Pull request closed without merging**